src/constants/request.ts

```diff
@@ -40,9 +40,10 @@ export const URLs = {
   },
   courses: {
     get: '/courses',
+    getById: '/courses/:id',
     delete: '/courses',
     create: '/courses',
-    patch: '/courses'
+    patch: '/courses/:id'
```

Comment:
I would rename it to "update", but it's up to you.
```suggestion
    update: '/courses/:id'
```

Reply:
Personally, I think patch is better here, because update could also mean a PUT method. This way, it avoids confusion in the services.